.github/workflows/release.yml

```diff
@@ -8,6 +8,7 @@ jobs:
   release-crates-io:
     name: Release crates.io
     runs-on: ubuntu-latest
+    if: "startsWith(github.ref, 'refs/tags/')"
```

Comment:
This isn't necessary but it makes it simpler to test this workflow by simply adding a `on.branches` condition in L4.